core/src/io/anuke/mindustry/ui/dialogs/CustomGameDialog.java

```diff
@@ -155,4 +158,89 @@ private void displayGameModeHelp(){
         d.show();
     }
 
-}
+    private class CustomRulesDialog extends FloatingDialog{
+        private Table main;
+
+        public CustomRulesDialog(){
+            super("$mode.custom");
+
+            setFillParent(true);
+            shown(this::setup);
+            addCloseButton();
+        }
+
+        void setup(){
+            cont.clear();
+            cont.pane(m -> main = m);
+            main.margin(10f);
+            main.setWidth(1000f);
```

Comment:
Setting the width of an element that participates in layout doesn't (or shouldn't) do anything. Even if it did, this is far too wide for a mobile device to fit on their screen in portrait mode.